components/atom/input/src/Input/Component/index.scss

```diff
@@ -38,6 +38,8 @@ $class-read-only: '#{$base-class}--readOnly';
     &:not(#{$class-read-only}) {
       background: $bgc-atom-input-disabled;
     }
+    opacity: 1;
+    -webkit-appearance: none;
```

Comment:
did you try to just add `appearance: none;` and let postCSS do the magic?

Reply:
Agree

Reply:
thought of doing that but tbh I wasn't sure how we're dealing with autoprefixer and since the issue is only occurring on Safari I just left it with the -webkit- prefix.

Reply:
if you just add the regular declaration, then every vertical can choose which browsers want to prefix by configuring its bundler. I believe this is more aligned with the SUI philosophy.

Reply:
gotcha :)